src/pages/LandingPage/FeatureCard.tsx

```diff
@@ -1,52 +1,61 @@
-import { Typography } from '@mui/material'
-import { styled } from '@mui/system'
-import React from 'react'
-import theme from '../../shared/theme'
+import { Typography } from "@mui/material";
+import { styled } from "@mui/system";
+import React from "react";
+import theme from "../../shared/theme";
 
 interface featureInfo {
-    title: string;
-    img: any,
-    description: string,
+  title: string;
+  img: any;
+  description: string;
 }
 
-const Feature = styled('div')({
-    height: '70%',
-    width: 300,
-    borderRadius: 15,
-    display: 'flex',
-    position: 'relative',
-    flexDirection: 'column',
-    alignItems: 'center',
-    justifyContent: 'center',
-    background: 'linear-gradient(to bottom, rgba(255, 255, 255, 0.1), transparent)',
-    borderTop: '5px solid ' + theme.palette.primary.main,
-})
+const Feature = styled("div")({
+  height: "70%",
+  width: 300,
+  borderRadius: 15,
+  display: "flex",
+  position: "relative",
+  flexDirection: "column",
+  alignItems: "center",
+  justifyContent: "center",
+  background:
+    "linear-gradient(to bottom, rgba(255, 255, 255, 0.1), transparent)",
```

Comment:
nit: Create a variable for this color

Reply:
this one I'll leave to @digitdustin because I don't know if this is already part of the theme